examples/spark/src/main/scala/org/apache/carbondata/examples/CaseClassDataFrameAPIExample.scala

```diff
@@ -42,8 +45,25 @@ object CaseClassDataFrameAPIExample {
       .mode(SaveMode.Overwrite)
       .save()
 
+    val employee = List(Employee("anubhav", 20000))
```

Comment:
@anubhav100 @jackylk  I am new to carbondata, but any reason for not adding the precision as below? 

case dt: DecimalType => s"${CarbonType.DECIMAL.getName}(${dt.precision}, ${dt.scale})